client/src/modules/patients/registry/modals/mergePatients.modal.js

```diff
@@ -27,6 +27,11 @@ function MergePatientsModalController(Patients, $state, Notify, Instance) {
   function submit() {
     if (vm.allAreEmployees) { return null; }
 
+    // Quick fix prevent fatar error : 2020-07-07 By: lomamech
```

Comment:
I realized that for the old module, merge of patiens, that when we do not select one and that validated, that abruptly stopped the server

Reply:
My English is not correct because i use Google Translate 
 ![image](https://user-images.githubusercontent.com/8035588/86894089-7b3a5980-c0fa-11ea-9bdf-f7794f35872a.png)

Reply:
I always put my French into Google translate to get the accents correct 😆